### PR TITLE
utils_test.libvirt: workaround to delete external disk snapshot.

### PR DESCRIPTION
**Pull request by rbian**
The command cause 'Leaked clusters were noticed during image check'
error in the test.

### CODE REVIEW

**Comment by rbian** [comment]
This solution is in testing, please don't merge.

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -488,7 +509,6 @@ def setup_or_cleanup_iscsi(is_setup, is_login=True,
         _iscsi.emulated_id = _iscsi.get_target_id()
         _iscsi.cleanup()
         process.run("rm -f %s" % emulated_path)
-        process.run("vgscan --cache", ignore_status=True)
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: Just curious, I'm not clear why we have this command, and now you delete it.

Comment (rbian) [inline virttest/utils_test/libvirt.py]: I am not sure why we use this command here, but it really caused some problems.
2016-04-15 01:47:28,988 process          L0331 INFO | Running 'vgscan --cache'
2016-04-15 01:47:28,993 process          L0418 DEBUG| [stderr] File descriptor 3 (/root/avocado/job-results/job-2016-04-15T01.46-bb44a47/job.log) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 4 (pipe:[388333]) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 5 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 6 (pipe:[388333]) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 7 (/dev/null) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 8 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python
2016-04-15 01:47:28,994 process          L0418 DEBUG| [stderr] File descriptor 9 (/dev/urandom) leaked on vgscan invocation. Parent PID 6794: python

**Comment by will-Do** [comment]
ACK

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (waynesun09) [inline virttest/utils_test/libvirt.py]: vgscan is introduced in commit 2aadb5b

**Comment by waynesun09** [comment]
since your testing is working well without vgscan, ACK

**Pull request merged**